libewf/libewf_single_files.c

```diff
@@ -1604,6 +4538,26 @@ int libewf_single_files_parse_file_entry(
 			else if( ( type_string[ 0 ] == (uint8_t) 'p' )
 			      && ( type_string[ 1 ] == (uint8_t) 'm' ) )
 			{
+				if (value_string_size > 0 && *value_string == '-')
```

Comment:
To my knowledge only `-1` is a known exception value. @jtsylve do you have insight into more exception values?

Reply:
Yes - as far as I'm aware, -1 is the only value that we'd expect for an item that doesn't have permissions.